src/ui/printer.rs

```diff
@@ -135,7 +135,13 @@ impl<W: Write> Printer<W> {
                     }
                 }
 
+                // Disable line wrapping
+                write!(self.writer, "\x1B[?7l")?;
+
                 write!(self.writer, "{buf}")?;
+
+                // Re-enable line wrapping
+                write!(self.writer, "\x1B[?7h")?;
```

Comment:
@spenserblack 

I think it's fairly safe to do it like this. My concern was to not leave the user's terminal in an altered state in case of an error with the line wrap still enabled. But with this diff the only scenario in which it could happen is if the program panics at line 141.

What do you think ?

An easy alternative would be to implement the `Drop` trait to `Printer<W>` in which we would add `write!(self.writer, "\x1B[?7h")?;` but it feels over the top 🤔

Reply:
No, this looks good to me! Since `Drop::drop` doesn't return a `Result`, the call to `write!` would have to unwrap, right? Probably best to keep potentially panicking code out of `drop` :slightly_smiling_face:

Any reason not to make it one call to `write!` instead of 3?

Reply:
>No, this looks good to me! Since Drop::drop doesn't return a Result, the call to write! would have to unwrap, right? Probably best to keep potentially panicking code out of drop 🙂

Very well, that reassures me 😩 

>Any reason not to make it one call to write! instead of 3?

You're right it's better if we merge them into one -> ddba00b